site/components/map/ServicesMap.tsx

```diff
@@ -235,6 +245,26 @@ const Map = ({
         }
     }, [features]);
 
+    useEffect(() => {
+        if (markerData && markerData.length > 0 && services) {
```

Comment:
I think it would be better if we handle retrieving the service routes when a user clicks select all rather than on drawing the polygon, it's quite an expensive query so we don't want to make it every time the user tweaks a polygon

Reply:
done